pennylane/devices/default_tensor.py

```diff
@@ -172,7 +172,8 @@ class DefaultTensor(Device):
         contract (str): The contraction method for applying gates in the MPS method. It can be either ``auto-mps`` or ``nonlocal``.
             ``nonlocal`` turns each gate into a Matrix Product Operator (MPO) and applies it directly to the MPS,
             while ``auto-mps`` swaps nonlocal qubits in 2-qubit gates to be next to each other before applying the gate,
-            then swaps them back. Default is ``auto-mps``.
+            then swaps them back. Default is ``auto-mps``. TODO: update description with new choices
```

Comment:
just a fly-by comment: also TODO: update code example in details section to set method

Reply:
@Qottmann Thanks Korbinian, I know that part should be updated as well. Please keep in mind that this PR is still a draft (under development) and not ready for review. When the time comes, if you want to be a reviewer I'll tag you :+1: . Thanks again!